File: CS/TestConditionalAppearanceRules.UnitTests/TestApplication.cs
using DevExpress.ExpressApp.Xpo;
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Layout;

namespace TestConditionalAppearanceRules.UnitTests {
    class TestApplication : XafApplication {
        protected override LayoutManager CreateLayoutManagerCore(bool simple) {
            return null;
        }
    }
}

File: VB/TestConditionalAppearanceRules.UnitTests/TestApplication.vb
Imports DevExpress.ExpressApp.Xpo
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Layout

Namespace TestConditionalAppearanceRules.UnitTests
    Friend Class TestApplication
        Inherits XafApplication

        Protected Overrides Function CreateLayoutManagerCore(ByVal simple As Boolean) As LayoutManager
            Return Nothing
        End Function
    End Class
End Namespace
